jcl/src/java.base/share/classes/java/lang/invoke/MethodHandleResolver.java

```diff
@@ -56,6 +60,10 @@ final class MethodHandleResolver {
 	private static final int BSM_TYPE_ARGUMENT_INDEX = 2;
 	private static final int BSM_OPTIONAL_ARGUMENTS_START_INDEX = 3;
 
+/*[IF JAVA_SPEC_VERSION >= 16]*/
+	private static final String INVOKE_EXACT = "invokeExact"; //$NON-NLS-1$
```

Comment:
What is the value of this field rather than just using the String directly? It only has one reader.

Reply:
Removed this field (which was used previously in debugging) given it is only used in `upcallLinkCallerMethod`.